Assets/scripts/LoadCatalog.cs

```diff
@@ -275,6 +372,174 @@ private void ChangeContentToCategory(string category)
         }        
     }
 
+    private void ChangeContentToBrand(string brand)
+    {
+        GameObject content = GameObject.Find("Content");
+
+        hideListings();
+
+        foreach (Item itemInList in loadedItems)
+        {
+            if (itemInList.GetBrand().Contains(brand))
+            {
+                showListing(itemInList);
+                content.transform.Find($"Listing: {itemInList.GetItemID()} {itemInList.GetName()}").gameObject.SetActive(true);
+            }
+        }
+    }
+
+    private void ChangeContentToDesigner(string designer)
+    {
+        GameObject content = GameObject.Find("Content");
+
+        hideListings();
+
+        foreach (Item itemInList in loadedItems)
```

Comment:
I think the variable should only be item not itemInList because it's kinda obvious when read the code as for each item in loadedItems.

Reply:
Ok got it. Will push another commit sometime later today with the variables renamed :-)